universal-application-tool-0.0.1/app/views/components/SelectWithLabel.java

```diff
@@ -0,0 +1,59 @@
+package views.components;
+
+import static j2html.TagCreator.option;
+import static j2html.TagCreator.select;
+
+import com.google.common.collect.ImmutableList;
+import j2html.attributes.Attr;
+import j2html.tags.ContainerTag;
+import j2html.tags.Tag;
+import java.util.AbstractMap.SimpleEntry;
+
+public class SelectWithLabel extends FieldWithLabel {
+
+  private ImmutableList<SimpleEntry<String, String>> options = ImmutableList.of();
+
+  public SelectWithLabel(String inputId) {
+    super(select(), inputId);
+  }
+
+  public SelectWithLabel setOptions(ImmutableList<SimpleEntry<String, String>> options) {
+    if (this.isRendered) {
```

Comment:
Question (blocking): should this throw instead of silently being a no-op?

Reply:
Discussed with Bion offline. Changing the code a little so that we can create similar Fields from the same "builder."